ProcessLib/GroundwaterFlowProcess.h

```diff
@@ -316,6 +316,18 @@ class GroundwaterFlowProcess : public Process
         delete _mesh_subset_all_nodes;
     }
 
+    /// Explicitly release memory of global vector, matrix and linear solvers
+    /// for MPI based parallel computing
+    void releaseEquationMemory()
+    {
+    // #ifdef USE_PETSC
+    //     delete _A.release();
```

Comment:
If `releaseEquationMemory()` is called before the process is destroyed, then maybe these delete statements should stay, such that memory will really be released. Otherwise: why don't you remove these lines?